transcom/mymove
# MB-12186 remove access code

## [Jira ticket](https://dp3.atlassian.net/browse/MB-12186) for this change

## Summary

The original intent of this work was to refactor validation patterns, but it became apparent Access Code is a deprecated service. So this work removes it and references from the code base. To help in reviewing the changes structured the commits into sections. 

## Setup to Run Your Code

`make db_dev_reset db_dev_migrate`
`make server_run`
`make client_run`

## Verification Steps for Reviewers

There's not a lot to check here, as it's a full removal. To be safe create and manage a service member from the office app. Also a second check to make sure I didn't miss any references in the code base is welcome. 

### Database

#### Any new migrations/schema changes:
